inginious/frontend/static/js/admin.js

```diff
@@ -0,0 +1,79 @@
+//
+// This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+// more information about the licensing of this file.
+//
+function revoke_binding(event){
+    var username = $(this).data("username");
+    var binding_id = $(this).data("binding");
+    $.post({
+      url: "user_action",
+      data: {
+        "username":username,
+        "action":"revoke_binding",
+        "binding_id": binding_id
+      },
+    }).done(function(feedback) {
+        if (!feedback.hasOwnProperty('error')){
+            window.location.href = "users";
+        }
+        $('#feedback_bindings').text(feedback['message']);
```

Comment:
Put this in an ``else`` statement. Otherwise, they are executed before the redirection is effective.